justfile

```diff
@@ -213,39 +220,32 @@ bootstrap *ARGS:
     pipx run perflint ../djangopackages-git/ --load-plugins=perflint
 
 # Compile new python dependencies
-@pip-compile *ARGS:
+@lock *ARGS:
     docker-compose run \
         --entrypoint= \
         --rm django \
             bash -c "uv pip compile {{ ARGS }} requirements.in \
                     --generate-hashes \
-                    --resolver=backtracking \
                     --output-file requirements.txt"
 
     docker-compose run \
         --entrypoint= \
         --rm django \
             bash -c "uv pip compile {{ ARGS }} docs/requirements.in \
                     --generate-hashes \
-                    --resolver=backtracking \
                     --output-file docs/requirements.txt"
 
-# Upgrade existing Python dependencies to their latest versions
-@pip-compile-upgrade:
-    just pip-compile --upgrade
-
 # Run pre-commit
 @pre-commit *ARGS:
     pre-commit run {{ ARGS }}
 
 @pre-commit-all-files:
     just pre-commit --all-files
 
-# TODO: Make the target-version a variable
-
-# Run `django-upgrade` with a target version of 4.1
+# Upgrade existing Python dependencies to their latest versions
 @upgrade:
-    git ls-files -- "*.py" | xARGS django-upgrade --target-version=4.1
+    just lock --upgrade
+    # git ls-files -- "*.py" | xARGS django-upgrade --target-version=4.1
```

Comment:
Target version 5.0?

Reply:
Yes, but I think this was carry over from before pre-commit. I'll clean it up in a follow up PR since it should run when we lint.